src/main/java/Duke.java

```diff
@@ -11,46 +11,68 @@ public static void main(String[] args) {
         // Greeting Message
         System.out.println("Hello, I'm Duke!");
         System.out.println("What can I do for you?");
-        System.out.println("\n");
 
         Scanner sc = new Scanner(System.in);
         int inputs = 0;
```

Comment:
change "inputs" to "input".
plural form only used for collection of variables/objects